apis/cluster/composition.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     kind: XCluster
   resources:
     - base:
-        apiVersion: aws.platformref.upbound.io/v1alpha1
+        apiVersion: aws.platform.upbound.io/v1alpha1
```

Comment:
@ytsarev @haarchri I don't know whether we're targeting backward compatibility with this reference platform (which I can understand if we don't), but wanted to share that I am getting the following when I want to upgrade from `v0.6.0` to the `latest master` because of this change:

```
  Warning  ComposeResources  45s (x316 over 5h11m)  defined/compositeresourcedefinition.apiextensions.crossplane.io  cannot compose resources: cannot parse base template of composed resource "compositeNetworkEKS": cannot change the kind of a composed resource from aws.platformref.upbound.io/v1alpha1, Kind=XNetwork to aws.platform.upbound.io/v1alpha1, Kind=XNetwork (possible composed resource template mismatch)
```

Reply:
We're going to simplify all aspects of this platform reference into single configurations, making it easier to build high-level abstractions. Also, we'll be changing `platform-ref ` to simply `platform`. We'll include some documentation about when we'll start creating the next release, but that will take a few more days. Thanks!